### DISCUSSION TITLE
ADD QUANTITY WITH DECIMAL IN POS SCREEN

### DISCUSSION THREAD

**Discussion by ayoub-hs** [Blair2004/NexoPOS#561] [Q&A]
HOW CAN WE ADD PRODUCT QUANTITY WITH DECIMAL IN POS SCREEN WHEN THE NUMPAD CONTAINS ONLY NUMBERS WITHOUT "." FOR DECIMAL 
FOR EXAMPLE I HAVE A PRODUCT 'OLIVE OIL' WITH UNIT 'LITRE' , I WANT TO SELL '1.5L' BUT CAN'T ENTER THAT IN NUMPAD

**Comment by Blair2004** [comment]
This will be possible with the unit of measure module. Without adding a support for decimal.
